creator-node/src/index.js

```diff
@@ -71,8 +72,9 @@ const startApp = async () => {
 
     await serviceRegistry.initServices()
     logger.info('Initialized services!')
-
     appInfo = initializeApp(config.get('port'), serviceRegistry)
+
+    await pinCID(serviceRegistry.ipfsLatest)
```

Comment:
I don't think we should prevent startup if this operation is unable to be performed - it has no user ramifications and is dashboard only. Pin operations can actually hang for quite some time and If this is slow for whatever reason or is unable to complete we should still proceed with startup. Is this an intentional behavior?

Reply:
I don't think this prevents app startup, does it?  
I think the server is started a couple lines above this in `initializeApp` 
Not sure if there are other processes/jobs that wait on this startApp function though. 
I actually wasn't 100% sure where to put this in the creator-node project, lmk if there's somewhere better.

Reply:
you're totally right we don't actually await on the app getting started anywhere